nats-streaming-server.go

```diff
@@ -127,7 +127,7 @@ func parseFlags() (*stand.Options, *natsd.Options) {
 	flag.StringVar(&stanOpts.ClientCert, "tls_client_cert", "", "Path to a client certificate file")
 	flag.StringVar(&stanOpts.ClientKey, "tls_client_key", "", "Path to a client key file")
 	flag.StringVar(&stanOpts.ClientCA, "tls_client_cacert", "", "Path to a client CA file")
-	flag.BoolVar(&stanOpts.EmbedNATS, "embed", true, "Embed the NATS Server")
+	flag.BoolVar(&stanOpts.EmbedNATS, "embed_nats", stand.DefaultEmbedNATS, "Embed the NATS Server")
```

Comment:
That wasn't what I meant by "two". I misunderstood your opening comment in
the PR, but I get it now.

Regarding making the user responsible for providing user/pwd in URL, it
seems we would have to do so, no?

Another question would be: will any of this become more confusing when
clustering is introduced for NATS Streaming?

On Thu, Jul 14, 2016 at 10:02 AM, Ivan Kozlovic <notifications@github.com>
wrote:

> In nats-streaming-server.go
> <https://github.com/nats-io/nats-streaming-server/pull/125#discussion_r70833016>
> :
>
> > @@ -127,7 +127,7 @@ func parseFlags() (*stand.Options, *natsd.Options) {
> >  	flag.StringVar(&stanOpts.ClientCert, "tls_client_cert", "", "Path to a client certificate file")
> >  	flag.StringVar(&stanOpts.ClientKey, "tls_client_key", "", "Path to a client key file")
> >  	flag.StringVar(&stanOpts.ClientCA, "tls_client_cacert", "", "Path to a client CA file")
> > -	flag.BoolVar(&stanOpts.EmbedNATS, "embed", true, "Embed the NATS Server")
> > +	flag.BoolVar(&stanOpts.EmbedNATS, "embed_nats", stand.DefaultEmbedNATS, "Embed the NATS Server")
>
> There was only one (it was renamed, this is why you see 2 lines ;-) ).
> But, no problem, I can indeed replace this with -server and the user
> would provide the url to the external NATS Server. Would the user be
> responsible from providing the user/pwd in that URL or not? The reason I
> ask is that currently we build the URL based on the provided NATS options.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/nats-io/nats-streaming-server/pull/125/files/a1d758fd084007e2f30e194bad2f7525e8236061..1b7ca0d88480c871cf6e334a647ed6194844fc21#r70833016>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGq4LLFlLJhQD_EmF9cvojFDy1g_0qeWks5qVl2dgaJpZM4JLmMr>
> .
>



-- 
*Larry McQueary* | Director, Messaging Technology
larry@apcera.com | @mcqueary | github.com/mcqueary